### DISCUSSION TITLE
Roadmap needs updating, Q4 2022

### DISCUSSION THREAD

**Discussion by davidtong** [microsoft/fluentui#25395] [Ideas] [answered]
The current roadmap stops at Q2 2022: https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-Roadmap

Would be nice to know what's coming up in Q4 and H1 2023.

**Comment by paulgildea** [comment, marked as answer]
Thanks @davidtong - for the time being we need to go back and update that page. Thanks for keeping us honest 😊. A majority of our work is around Component build out and you can find more information on this wiki page: https://github.com/microsoft/fluentui/wiki/Fluent-UI-React-v9-Component-Roadmap

Thanks for pulling me in @khmakoto.

Hope that helps!
